docs/source/custom_directives.py

```diff
@@ -121,7 +121,7 @@ class SupportedProperties(BaseShield):
     required_arguments = 1
     final_argument_whitespace = True
 
-    def run(self) -> List[nodes.Node]:
+    def run(self, params, alt, _) -> List[nodes.Node]:
```

Comment:
Why these changes?

Reply:
This is consistent with the base class signature. run is overriden method.